ui/src/app/enums/browsers.ts

```diff
@@ -0,0 +1,3 @@
+export enum Browsers {
```

Comment:
have a index file for enums/index.ts and export all the enum files there. So, we would have single point interface for enums

Reply:
Done added index file